src/test/java/com/google/api/codegen/protoannotations/testdata/java_library.baseline

```diff
@@ -3443,17 +3442,15 @@ public class TuringProgCallableStreamingBidi {
       BidiStream<DiscussBookRequest, Comment> bidiStream =
           libraryClient.discussBookCallable().call();
 
-      // String imageFileName = "image_file.jpg";
+      // imageFileName = "image_file.jpg";
       BookName name = ShelfBookName.of("[BOOK_SHELF]", "[BOOK]");
       Path path = Paths.get("comment_file");
-      byte[] data = Files.readAllBytes(path);
-      ByteString comment = ByteString.copyFrom(data);
+      byte[] data = Files.readAllBytes(path);ByteString comment = ByteString.copyFrom(data);
       Comment comment2 = Comment.newBuilder()
         .setComment(comment)
         .build();
       path = Paths.get(imageFileName);
-      data = Files.readAllBytes(path);
-      ByteString image = ByteString.copyFrom(data);
+      data = Files.readAllBytes(path);ByteString image = ByteString.copyFrom(data);
```

Comment:
It might look prettier if we kept the `data = ...` and `ByteString ...` lines separate

Reply:
Thanks for the catch! It's really strange that whenever I wrapper something with an `@if` block I have to add a new blank line to preserve the original output.